### PR TITLE
add messageformat v0.2.2

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/SlexAxton/messageformat.js
**Watch 42
  Star 881
 Fork 81**
@Amomo could u help me check this PR for #5870 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/messageformat/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "MIT",
+    "CLA"
+  ],
+  "homepage": "https://github.com/SlexAxton/messageformat.js",
+  "npmName": "messageformat",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "messageformat*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/messageformat/package.json]: What about `locale/`?

**Comment by maruilian11** [comment]
ping @Amomo

**Comment by maruilian11** [comment]
ping @Amomo

**Comment by Amomo** [comment]
@maruilian11 sorry for late. 😱 **messageformat** seems has `v0.3.1` now, could you update this PR?
Thank you very much!

File: ajax/libs/messageformat/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "nopt": "~2.1.2",
+    "coffee-script": "~1.7.0",
+    "watchr": "~2.4.9",
+    "underscore": "~1.5.2",
+    "async": "~0.2.10",
+    "glob": "~3.2.8"
+  },
+  "licenses": [
+    "MIT",
+    "CLA"
```
Comment (PeterDaveHello) [inline ajax/libs/messageformat/package.json]: What's the CLA license?

File: ajax/libs/messageformat/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "name": "messageformat",
+  "version": "0.2.2",
+  "author": {
+    "name": "Alex Sexton",
+    "email": "alexsexton@gmail.com"
+  },
+  "description": "PluralFormat and SelectFormat Message and i18n Tool - A JavaScript Implemenation of the ICU standards.",
+  "repository": {
+    "type": "git",
+    "url": "https://SlexAxton@github.com/SlexAxton/messageformat.js.git"
```
Comment (PeterDaveHello) [inline ajax/libs/messageformat/package.json]: We don't need the `SlexAxton@`, do we?

File: ajax/libs/messageformat/package.json (hunk 2 above)
Comment (maruilian11) [inline ajax/libs/messageformat/package.json]: @PeterDaveHello 
https://github.com/SlexAxton/messageformat.js/blob/0.2.2/package.json#L44
http://dojofoundation.org/about/cla

Comment (PeterDaveHello) [inline ajax/libs/messageformat/package.json]: So why don't you add a reference url?

Comment (maruilian11) [inline ajax/libs/messageformat/package.json]: i will do it next time.

**Comment by maruilian11** [comment]
@Amomo it's ok, i have updated it~

**Comment by PeterDaveHello** [comment]
I wonder if we need `messageformat-parser.js`?

**Comment by maruilian11** [comment]
@Amomo could you help me check this PR?

**Comment by Amomo** [comment]
@maruilian11 could you also update the title of this PR? thanks~
@PeterDaveHello I think this PR is fine, could you take a look? thank you!

**Pull request merged**